JuliaInterface/example/function_perform.gi

```diff
@@ -0,0 +1,48 @@
+LoadPackage( "JuliaInterface" );
+
+dirs:= DirectoriesPackageLibrary( "JuliaInterface", "example" );
+
+JuliaIncludeFile( Filename( dirs, "function_perform.jl" ) );
+
+if not IsBound(_JuliaFunctionByModuleNoConv) then
+  _JuliaFunctionByModuleNoConv:= _JuliaFunctionByModule;
+  _JuliaFunctionNoConv:= _JuliaFunction;
```

Comment:
This `if` block is not really needed.

And beyond that, as I mentioned elsewhere, I'd really merge `_JuliaFunctionByModuleNoConv`, `_JuliaFunctionByModule`, `_JuliaFunctionNoConv`, `_JuliaFunction` into a single function.